intro-HPC/ch_MATLAB.tex

```diff
@@ -0,0 +1,160 @@
+\chapter{MATLAB}
+\label{ch:matlab}
+
+\section{Why is the MATLAB compiler required?}
+
+The main reason behind this alternative way of using MATLAB is licensing: only
+a limited number of MATLAB sessions can be active at the same time. However, once
+the MATLAB program is compiled using the MATLAB compiler, the resulting stand-alone
+executable can be run without needing to contact the license server.
+
+Note that a license is required for the MATLAB Compiler,
+see \url{https://nl.mathworks.com/help/compiler/index.html}. If the \verb|mcc|
+command is provided by the MATLAB installation you are using, the MATLAB compiler
+can be used as explained below.
+
+\ifgent
+Only a limited amount of MATLAB sessions can be active at the same time because
+there are only a limited amount of MATLAB research licences available on
+the \university MATLAB license server. If each job would need a license,
+licenses would quickly run out.
+\fi
+
+\section{How to compile MATLAB code}
+
+Compiling MATLAB code can only be done from the login nodes, because only login
+nodes can access the MATLAB license server, workernodes on clusters can not.
+
+To access the MATLAB compiler, the \verb|MATLAB| module should be loaded first. Make sure
+you are using the same \verb|MATLAB| version to compile and to run the compiled MATLAB
+program.
+
+\begin{prompt}
+%\shellcmd{module avail MATLAB}%
+----------------------/apps/%\sitename%/CO7/sandybridge/modules/all ----------------------
```

Comment:
the `CO7/sandybridge` is still site-specific, I would replace that with `%\defaultcluster%` (which is not correct strictly speaking, but it's good enough)